src/Sylius/Bundle/AdminBundle/Resources/views/Crud/Create/_content.html.twig

```diff
@@ -6,6 +6,8 @@
         {{ form_widget(form) }}
     {% endif %}
 
+    {{ sonata_block_render_event(eventPrefix ~ '.form', { 'resource': resource }) }}
```

Comment:
For me, it is consistent, we have slots: `after_sth` and `before_sth`, but also in some cases `form`, `header`, thanks to that, we know that sth is in a particular place and for example could add new field to form. `after_form` could be interpret that we are after form and we do not have possibility to add a new field.